mailbox/api/src/test/java/org/apache/james/mailbox/mock/MockMailboxSession.java

```diff
@@ -39,6 +39,7 @@
     private final static Random RANDOM = new Random();
 
     private final long sessionId = RANDOM.nextLong();
+    private SessionType type = SessionType.User;
```

Comment:
That's default value before. I don't want to break the old tests so I keep that as the default value. Is that OK?